### ISSUE TITLE
关于CAER-S数据集

### ISSUE THREAD

**Issue by myt889** [zengqunzhao/EfficientFace#27]
您好，我想咨询一下，在论文中提到The samples' number of the CAER-S dataset employed in our work should be: all (69,982 samples), training set (48,995 samples), and test set (20,987 samples). 那么如果使用RetinaFace裁剪CAER-S数据集中的人脸
![image](https://github.com/zengqunzhao/EfficientFace/assets/77277761/6c1526e2-c547-4411-a02b-3c7f788b12df)，如果一张图中有两张人脸，如何处理？只选一张人脸，还是选择两张？非常期待您的回复。

**Comment by zengqunzhao** [comment]
> 您好，我想咨询一下，在论文中提到The samples' number of the CAER-S dataset employed in our work should be: all (69,982 samples), training set (48,995 samples), and test set (20,987 samples). 那么如果使用RetinaFace裁剪CAER-S数据集中的人脸 ![image](https://private-user-images.githubusercontent.com/77277761/333395174-6c1526e2-c547-4411-a02b-3c7f788b12df.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4MTQ5MzMsIm5iZiI6MTcxNjgxNDYzMywicGF0aCI6Ii83NzI3Nzc2MS8zMzMzOTUxNzQtNmMxNTI2ZTItYzU0Ny00NDExLWEwMmItM2M3Zjc4OGIxMmRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDEyNTcxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1NzNlNmY1OTI5MzVjNmVmMDljMGU5ZjczN2RiYTJjM2IzOWU2OGYwYTc4NmFmN2EyM2JjYzI0Zjk2MjMwZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wDdUurHoyqPtiSoGH-WG2kbyEwjO9XZVkAXFIPVyXso)，如果一张图中有两张人脸，如何处理？只选一张人脸，还是选择两张？非常期待您的回复。

Hi, 这个数据集里面确实有一些多张脸的image，你可以根据人脸检测的置信度选一下，对于一张image里面存在多张高置信度的人脸情况，可能需要根据emotion label人工挑一下，不过这样的样本很少。希望对你有用！
